cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2122,10 +2122,19 @@
   </property>
 
   <property>
-    <name>metrics.data.table.retention.resolution.1.seconds</name>
+    <name>metrics.minimum.resolution</name>
+    <value>1</value>
+    <description>
+      Minimum metrics resolution in seconds, this value should be less than 60 seconds, otherwise the minimum metrics
+      resolution will be 60 seconds.
+    </description>
+  </property>
+
+  <property>
+    <name>metrics.data.table.retention.resolution.minimum.seconds</name>
     <value>7200</value>
     <description>
-      Retention resolution in seconds of the 1-second resolution table;
+      Retention resolution in seconds of the minimum resolution table;
```

Comment:
we should also mention that this retention will not be applied if min resolution is >=60